my-app/src/AddTodoForm.tsx

```diff
@@ -0,0 +1,33 @@
+import React, { useState } from 'react';
+
+interface Props {
+  addTodo: AddTodo;
+}
+
+export const AddTodoForm: React.FC<Props> = (param) => {
+  //const { addTodo } = param;
```

Comment:
Don't leave commented code, it's dirty :P 
If you leave it for documentation purposes, I'd advise you to put an actual comment with it.

Reply:
I shall make a documentation of it. I wanted to keep track of the possible destructuration here.